<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterPatientDialog</class>
 <widget class="QDialog" name="AjouterPatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ville : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de consultation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Durée de consultation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Priorité :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Téléphone :</string>
   </property>
  </widget>
  <widget class="QListView" name="ressourcesListView">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resources : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Commentaires : </string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="commentairesEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nomEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>&gt;A&lt;AAAAAAAAAAAAAAAAAAAAAAAA</string>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="placeholderText">
    <string>Nom</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenomEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>&gt;A&lt;AAAAAAAAAAAAAAAAAAAAAAAA</string>
   </property>
   <property name="placeholderText">
    <string>Prenom</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="adresseEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Adresse</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="villeEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>&gt;A&lt;AAAAAAAAAAAAAAAAAAAAAAAA</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Ville</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="telephoneEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>9999999999</string>
   </property>
   <property name="placeholderText">
    <string>06...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="codePostalEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>99999</string>
   </property>
   <property name="placeholderText">
    <string>37200</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="dureeConsultationEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>99</string>
   </property>
   <property name="text">
    <string>60</string>
   </property>
   <property name="placeholderText">
    <string>60</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateConsultationEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2017</year>
     <month>4</month>
     <day>17</day>
    </date>
   </property>
  </widget>
  <widget class="QRadioButton" name="priorite1Radio">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="priorite2Radio">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="priorite3Radio">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="priorite4Radio">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="priorite5Radio">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouterPatientButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annulerButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
